util/upload-instance-slot-map.py

```diff
@@ -43,6 +43,7 @@ def dump_instances(instance_details, region, aws_credentials=None):
         if "Compute Instance" in product.get("productFamily"):
             instance = product.get("attributes")
             instances[instance.get("instanceType")] = {"vcpus": instance.get("vcpu")}
+            instances[instance.get("instanceType")]["memory"] = instance.get("memory")
```

Comment:
minor: can you report somewhere in this file an extract from the initial doc used to read this data? What is the format for memory? is it in Gb and is it in the format we need in the daemons?

Reply:
Added comment